File: python/LineUp/root_command/lup.py
from ..interpreter.line import Line

class Lup:
    def __init__(self, interpreter):
        self.interpreter = interpreter

    def load_command(self, file):
        return self.execute(file)
    
    def execute(self, file):
        for i in open(file).read().split(";"):
            data = Line(" ".join(i.split("\n"))).execute(self.interpreter.global_variable, self.interpreter.global_class)
            if isinstance(data, list) and data[0] == "/L_e*/":
                return data[1]
    

File: LineUp/php/LineUp/root_command/Lup.php
<?php

/* It executes a file. */
class Lup
{
    public $interpreter;

    public function __construct(&$interpreter)
    {
        $this->interpreter = $interpreter;
    }

    public function load_command($command_name, $parameters)
    {
        return $this->execute($command_name);
    }

    private function execute($file)
    {
        foreach (explode(";", file_get_contents($file)) as $value) {
            $data = (new Line(
                    implode(" ", explode(PHP_EOL, $value))
                )
            )->execute(
                $this->interpreter->global_variable,
                $this->interpreter->global_class
            );
            if (is_array($data) && $data[0] == "/L_e*/") {
                return $data[1];
            }
        }
    }
}
